code/game/gamemodes/marker/gamemodes.dm

```diff
@@ -116,10 +116,14 @@ GLOBAL_DATUM_INIT(shipsystem, /datum/ship_subsystems, new)
 	set category = "Admin"
 	set desc = "Forces the marker to immediately activate"
 
-	var/confirm = alert(src, "You will be activating the marker. Are you super duper sure?", "Make us Whole?", "Send in the Necromorphs!", "On second thought, maybe not...")
-	if(confirm != "Send in the Necromorphs!")
+	if(check_rights(R_MOD))
```

Comment:
this is already an admin verb, you don't need additional rights checks in here. they're handled before the verb is even added to a mob

Reply:
Fairly sure it's not. It's shown to Mentors.

Reply:
then its in the wrong admin category, correct the category. rights checks still don't go within these functions